### PR TITLE
Add Travis support

### PR DESCRIPTION
**Pull request by abdonrd**
Also, I think can help @rictic to check the `polymer-linter`.

PS: Someone with permissions, needs to activate the repository in [Travis: /Polymer](https://travis-ci.org/Polymer).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - google-chrome-stable
+node_js:
+  - node
+  - 6
+cache: yarn
+install:
+  - yarn global add polymer-cli
+  - polymer install
+before_script:
+  - polymer lint
```
Comment (rictic) [inline .travis.yml]: I'd leave this out here as it's likely to fail at head. I'll add it in as part of the PR or PRs that get things linting clean.

Comment (abdonrd) [inline .travis.yml]: Commented linting from now!

**Comment by abdonrd** [comment]
Friendly ping to @frankiefu :)
I think you have Travis permissions.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+language: node_js
+dist: trusty
+sudo: required
+addons:
+  firefox: latest
+  apt:
```
Comment (stramel) [inline .travis.yml]: Just use `chrome: stable` instead?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+      - google-chrome-stable
+node_js:
+  - node
+  - 6
+cache: yarn
+install:
+  - yarn global add polymer-cli
+  - polymer install
+before_script:
+  # - polymer lint
```
Comment (stramel) [inline .travis.yml]: I vote we enable it even with all the warnings to draw attention to them.

**Comment by abdonrd** [comment]
Updated to fix merge conflicts.

**Comment by abdonrd** [comment]
Closing very old PR...

**Pull request closed without merging**